source/MRVoxels/MRVoxelsFwd.h

```diff
@@ -24,12 +24,14 @@ class ObjectVoxels;
 struct OpenVdbFloatGrid;
 using FloatGrid = std::shared_ptr<OpenVdbFloatGrid>;
 
-MR_CANONICAL_TYPEDEFS( (template <typename T> struct), VoxelsVolumeMinMax,
+MR_CANONICAL_TYPEDEFS( ( template <typename T> struct ), VoxelsVolumeMinMax,
     ( SimpleVolumeMinMax, VoxelsVolumeMinMax<std::vector<float>> )
     ( SimpleVolumeMinMaxU16, VoxelsVolumeMinMax<std::vector<uint16_t>> )
     ( VdbVolume, VoxelsVolumeMinMax<FloatGrid> )
 )
 
+using VdbVolumes = std::vector<VdbVolume>;
```

Comment:
where is it used?

Reply:
In MRVoxelsLoad for auto casting

MR_VECTOR_LIKE_IMPL( VdbVolumes, VdbVolume )